joschahenningsen/TUM-Live
# playlist scroll pos

### Motivation and Context
If the playlist sidebar is not visible on page start, the scroll position is not set to the active item.

https://github.com/TUM-Dev/gocast/assets/17506411/ab908c9c-60e0-4656-8336-c8e4f9803a56

### Description
This pr adds a observer that scrolls to the right position upon sidebar open.

### Steps for Testing
Prerequisites:
- 1 Students

1. Log in
2. Navigate to a Livestream
3. Check if its scrolled to the right item if you open the playlist sidebar

### Screenshots

https://github.com/TUM-Dev/gocast/assets/17506411/f96f7a1a-ed2c-472e-a974-6197e7dd8f89
